ui/apps/platform/src/Containers/Integrations/utils/tableColumnDescriptor.ts

```diff
@@ -1,30 +1,39 @@
-import React from 'react';
-
-import { defaultColumnClassName } from 'Components/Table';
+import { BaseBackupIntegration } from 'types/externalBackup.proto';
+import { BaseImageIntegration } from 'types/imageIntegration.proto';
+import {
+    AuthPluginType,
+    AuthProviderType,
+    BackupIntegrationType,
+    BaseIntegration,
+    ImageIntegrationType,
+    NotifierIntegrationType,
+    SignatureIntegrationType,
+} from 'types/integration';
+import { SumoLogicNotifierIntegration, SyslogNotifierIntegration } from 'types/notifier.proto';
+import { SignatureIntegration } from 'types/signatureIntegration.proto';
 import { knownBackendFlags } from 'utils/featureFlags';
 
 import { daysOfWeek, timesOfDay } from './integrationUtils';
 
-const classNameWordBreak = `${defaultColumnClassName} word-break`;
```

Comment:
Why are we removing this className?

Reply:
Good question. Leftover Tailwind style. Code to render table cell does not refer to `className` prop:
* Th: https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Integrations/IntegrationsListPage/IntegrationsTable.tsx#L137-L143
* Td: https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Integrations/IntegrationsListPage/IntegrationsTable.tsx#L182-L203
* TableCellValue: https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Components/TableCellValue/TableCellValue.tsx#L12-L20